temp.py

```diff
@@ -0,0 +1,8 @@
+def reset_files(filepath, file_type="txt"):
+    with open(filepath, "w") as output_file:
+        if file_type == "txt":
+            output_file.write("")
+        elif file_type == "json":
```

Comment:
Implement a more robust file type handling mechanism, potentially using MIME types or a dedicated library to support more file types in the future. [medium]